base/doc/ltnews41.tex

```diff
@@ -640,6 +640,24 @@ \subsection{Correct the float placement algorithm}
 \githubissue{1645}
 
 
+\subsection{Correct \cs{CheckEncodingSubset}}
+
+In \cite{41:ltnews36} and again in \cite{41:ltnews39} we suggested
+that font maintainers should place an appropriate
+\cs{DeclareEncodingSubset} declaration into each \texttt{ts1\meta{family}.fd}
+file so that it is tied to the font definition and available if a font
+family is explicitly selected through
+\cs{fontfamily}\texttt{\{\meta{name}\}} instead of using some font
+support package. Unfortunately, doing this could result in incorrectly
+selected glyphs when the font encoding setting was evaluated before
```

Comment:
Not quite sure exactly what is meant here, maybe:

. . . glyphs if the font encoding got set before . . .

Reply:
well, neither it is just rubbish ... good catch.

should be   if the font subencoding setting was checked before ...

Reply:
So why did this get approved by so many other reviewers?